config/sync/user.role.news_supervisor.yml

```diff
@@ -13,4 +13,40 @@ permissions:
   - 'delete own news content'
   - 'edit any news content'
   - 'edit own news content'
+  - 'revert application revisions'
```

Comment:
Do News supervisor authors need edit permissions for all content types?

Reply:
Nope!